### ISSUE TITLE
BUG. Voiceovers & Stuff

### ISSUE THREAD

**Issue by Stesmus** [anidl/multi-downloader-nx#275]
### Program version

3.0.3

### Operating System

Windows

### Type

Cli

### Service

Crunchyroll

### Command used

anidl --service crunchy --srz GRVDN485R --all -q 0 --dubLang eng --dlsubs en --skipSubMux --fsRetryTime 5 --partsize 100 --fileName "${showTitle} [${season}-${episode}] [${height}p]"

### Show ID

GRVDN485R

### Episode

No matter

### Console Output

It doesn't matter. It's being downloaded.

### Additional Information

```Text
Hi. I specified the English voice acting, but Russian's downloaded at the output. And when downloading Russian, it doesn't download at all. If you have the opportunity check it, please.

P.S. There is another problem. When you download it completely with voiceovers, the subtitles are overwritten (full to forced, for example). I think it's necessary that the output should have the name of the voice acting, and not the general name (although some are downloaded with the name of the voice acting). In general, take a look. Maybe there is some way to fix it.

Thanks anyway.
```

**Comment by Stesmus** [comment]
Here's another problem. Doesn't see the names and downloads without it, sometimes overwrites the same file names.

=== Multi Downloader NX 3.0.3 ===

[Z:G6P8DV7V6] Yu-Gi-Oh! (Seasons: 6, EPs: 460) [SUB, DUB]
  [S:GR2P1NK0R] NO_TITLE (Season: 1) [DUB]
  [S:GR9P35GV6] Yu-Gi-Oh! Season 1 (Subtitled) (Season: 1) [SUB]
  [S:G6P8DVPK6] NO_TITLE (Season: 2) [DUB]
  [S:G6MGDMMV6] NO_TITLE (Season: 3) [DUB]
  [S:G6491J8WY] NO_TITLE (Season: 4) [DUB]
  [S:GYWEZV50Y] NO_TITLE (Season: 5) [DUB]

[Z:G62P107J6] Yu-Gi-Oh! 5D's (Seasons: 4, EPs: 277) [SUB, DUB]
  [S:GR8V1QN4R] NO_TITLE (Season: 1) [DUB]
  [S:G6JQV810R] Yu-Gi-Oh! 5D's Season 1 (Subtitled) (Season: 1) [SUB]
  [S:GR0913V0R] NO_TITLE (Season: 2) [DUB]
  [S:GY2P1D2KY] Yu-Gi-Oh! 5D's Season 2 (Subtitled) (Season: 2) [SUB]

[Z:GY8V1ED9Y] Yu-Gi-Oh! GX (Seasons: 4, EPs: 335) [SUB, DUB]
  [S:GRZX2843Y] NO_TITLE (Season: 1) [DUB]
  [S:GRJQVGPVY] Yu-Gi-Oh! GX (Subtitled) (Season: 1) [SUB]
  [S:GYGG4Q0WY] NO_TITLE (Season: 2) [DUB]
  [S:G6X01PZ5Y] NO_TITLE (Season: 3) [DUB]

**Comment by izu-co** [comment]
I think this should be fixed with the newest commit

**Comment by Stesmus** [comment]
I rechecked. Same.
